bundles/binding/org.openhab.binding.weather/src/main/java/org/openhab/binding/weather/internal/common/LocationConfig.java

```diff
@@ -145,12 +145,15 @@ public void setName(String name) {
      */
     public boolean isValid() {
         boolean valid = providerName != null && language != null && updateInterval != null && locationId != null;
+        if( !valid ) {
```

Comment:
Please use the formatter in the Eclipse IDE to comply, and also so this line doesn't get reformatted on the next commit.